pkg/kubewarden/components/PolicyServer/SysctlsArrayList.vue

```diff
@@ -82,44 +78,43 @@ export default {
       :add-allowed="true"
       :add-label="addLabel"
       :disabled="disabled"
-      :default-add-value="0"
+      :default-add-value="{ name: '', value: '' }"

```

Comment:
did arrayList change?

Reply:
I don't think it was correct to begin with actually 😅